### PR TITLE
feat: add submission copy to GraphQL schema

### PR DESCRIPTION
**Pull request by dariakoko**
As discussed in [this thread](https://artsy.slack.com/archives/D032AFMTJES/p1721828639408619), adding submission copies to MP
Addressed [ONYX-1135](https://artsyproduct.atlassian.net/browse/ONYX-1135) 

[ONYX-1135]: https://artsyproduct.atlassian.net/browse/ONYX-1135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by olerichter00 — APPROVED** [review]
Looks great! 🌟 

It would be great if we could also get the copy for listed submissions ([this case](https://github.com/artsy/force/blob/742930690d41d7119cd9848cd9b6917e6c4f2f69/src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASubmissionStatus.tsx#L226)). In Force I've done this by looking at the artwork's `listedArtworks` which should include at least one (published?) artwork in case the submission is already listed.

File: src/schema/v2/artwork/artworkConsignmentSubmissionType.ts [hunk 1]
```diff
@@ -86,28 +86,84 @@ const ArtworkConsignmentSubmissionType = new GraphQLObjectType<
...
+        type: GraphQLString,
+        resolve: ({ state }) => {
+          switch (state.toLowerCase()) {
+            case "rejected":
+              return "black60"
+            default:
+              return "black100"
+          }
+        },
+      },
```
Comment (olerichter00) [inline src/schema/v2/artwork/artworkConsignmentSubmissionType.ts:112]: What is the `stateLabelColor` for? Is it the button? Maybe for the button we could define the color like this:

```suggestion
      buttonVariant: {
        type: GraphQLString,
        resolve: ({ state }) => {
          switch (state.toLowerCase()) {
            case "approved":
            case "draft":
              return "primaryBlack"
            default:
              return "secondaryBlack"
          }
        },
      },
```

File: src/schema/v2/artwork/artworkConsignmentSubmissionType.ts [hunk 2]
```diff
@@ -86,28 +86,84 @@ const ArtworkConsignmentSubmissionType = new GraphQLObjectType<
...
+              return "Add Additional Information"
+            case "submitted":
+            case "published":
+              /* case "RESUBMITTED": */
+              return "Edit Submission"
+            default:
+              return null
+          }
+        },
+      },
```
Comment (olerichter00) [inline src/schema/v2/artwork/artworkConsignmentSubmissionType.ts:145]: Maybe we could define the color for the button like this:

```suggestion
      },
      buttonVariant: {
        type: GraphQLString,
        resolve: ({ state }) => {
          switch (state.toLowerCase()) {
            case "approved":
            case "draft":
              return "primaryBlack"
            default:
              return "secondaryBlack"
          }
        },
      },
```

File: src/schema/v2/artwork/artworkConsignmentSubmissionType.ts [hunk 3]
```diff
@@ -86,28 +86,84 @@ const ArtworkConsignmentSubmissionType = new GraphQLObjectType<
...
+            case "draft":
+              return "Complete Submission"
+            case "approved":
+              return "Complete Listing"
+            default:
+              return null
+          }
+        },
+      },
+      buttonLable: {
```
Comment (olerichter00) [inline src/schema/v2/artwork/artworkConsignmentSubmissionType.ts:128]:
```suggestion
      buttonLabel: {
```

File: src/schema/v2/artwork/artworkConsignmentSubmissionType.ts (hunk 1 above)
Comment (dariakoko) [reply src/schema/v2/artwork/artworkConsignmentSubmissionType.ts:112]: It's the color for `stateLabel`, it's black100 for all statuses and only for Submission Unsuccessful (rejected state) it's black60
Not sure if it's an overkill or not

**Comment by dariakoko** [comment]
Sure, we can add the copy for listed state as well @olerichter00 
Do we need to add another state in order to do that? I see we have `open: "Submission in progress"`, that looks like it could be the same as `listed`. Or, maybe I misused `published` state?

**Comment by dariakoko** [comment]
> Sure, we can add the copy for listed state as well @olerichter00 Do we need to add another state in order to do that? I see we have `open: "Submission in progress"`, that looks like it could be the same as `listed`. Or, maybe I misused `published` state?

I see [here](https://www.notion.so/artsy/Submission-Status-MyC-Artwork-7593508a62df447290deeeee34116899?pvs=4) that this is a new one. Solved ✅

**Review by nickskalkin — APPROVED** [review]
👍

**Pull request merged**